dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/OracleEvent.scala

```diff
@@ -239,10 +275,15 @@ object OracleEvent {
     */
   def verifyAttestations(
       announcement: OracleAnnouncementTLV,
-      attestations: Vector[SchnorrDigitalSignature],
+      attestationTLV: OracleAttestmentTLV,
       signingVersion: SigningVersion): Boolean = {
+    val attestations = attestationTLV.sigs
     val nonces = announcement.eventTLV.nonces
-    if (nonces.size != attestations.size || nonces != attestations.map(_.rx)) {
+    if (
+      announcement.publicKey != announcement.publicKey ||
```

Comment:
I think it's always `false`?

Reply:
Should we also use the attestation outcomes for verification? @nkohen

Reply:
Fixed to compare against correct key

Reply:
I don't think there's any reason we couldn't check the outcomes since they're there

Reply:
for enum that should just be a contains check and for multi-nonce just use the base and match against the regex for the correct size with the correct number of digits